tests.py

```diff
@@ -311,6 +311,28 @@ def test_import_from_qtwidgets():
     assert QPushButton.__name__ == "QPushButton", QPushButton
 
 
+def test_pyside2uic():
```

Comment:
I'm just testing pyside2uic generally. You can compile via it so I figured that was a good test. I'm **not** testing Qt.py's `--compile` or `--convert`.